src/main/kotlin/org/kamiblue/client/plugin/PluginError.kt

```diff
@@ -0,0 +1,47 @@
+package org.kamiblue.client.plugin
+
+import org.kamiblue.client.KamiMod
+import org.kamiblue.client.gui.mc.KamiGuiPluginError
+import org.kamiblue.client.util.Wrapper
+
+internal enum class PluginError {
+    HOT_RELOAD,
+    DUPLICATE,
+    UNSUPPORTED,
+    REQUIRED_PLUGIN;
+
+    fun handleError(loader: PluginLoader) {
+        val list = latestErrors ?: ArrayList<Pair<PluginLoader, PluginError>>().also { latestErrors = it }
+
+        when (this) {
+            HOT_RELOAD -> {
+                KamiMod.LOG.error("Plugin $loader cannot be hot reloaded.")
```

Comment:
Add `, restart your game to load this plugin`

Reply:
This only shows up in the log so just keep it short